doc/DataSlice.md

```diff
@@ -17,11 +17,14 @@ Below is the metadata list and a note explaining how they are calculated.
 
     * `total_rows` (`int`): total rows returned by a user query.
     * `total_returned_rows` (`int`): total of rows returned in a slice of results.
-    * `execution_time` (`float`): in milliseconds, query execution time.
-    * `conversion_time` (`float`): in milliseconds, time to convert results in a `pandas.DataFrame`.
+    * `query_generation_time` (`float`): in seconds, the time to modify/create a query to get just the slice and start the process to request external data.
+    * `data_extraction_time` (`float`): in seconds, the time to request the result of the modified query from external service. 
+    * `data_conversion_time` (`float`): in seconds, the time to convert results in a `pandas.DataFrame`.
+    * `permission_application_time` (`float`): in seconds, the time to filter datas with permissions.
+    * `stats_computed_time` (`float`): in seconds, the time to compute statistics and return data as result.
```

Comment:
In the model, it's called `compute_stats_time` https://github.com/ToucanToco/toucan-connectors/pull/546/files#diff-5feb095e781094a6657a509f814113ef04d1ecfefd4209765d89394564f57737R37
Which one do you want to keep?

(It's also `stats_computed_time` in the data service so I recommend keeping this one)

Reply:
👍🏾